client/webserver/site/src/js/wallets.ts

```diff
@@ -723,6 +726,13 @@ export default class WalletsPage extends BasePage {
       this.changeWalletPW = false
       this.setPWSettingViz(false)
     } else Doc.show(this.page.showChangePW)
+
+    // if wallet not loadable from file, hide file selector, else shows it.
+    if (walletNotLoadableFromFile.includes(walletDef.type)) {
+      Doc.hide(this.reconfigForm.fileSelector)
+    } else {
+      Doc.show(this.reconfigForm.fileSelector)
```

Comment:
I figured we'd do this above in the `if (walletDef.seeded || walletDef.type === 'token')` condition that we use to hide tint wait password form.
Is this conceptually different in some way that would require a separate whitelist?

Reply:
I don't believe so, the only reason I did not implemented it, inside the other if is to avoid nesting ifs, but I guess if it the wallet is seeded or token, it is enough for hiding the "load from file" link?

Reply:
At present, "seeded" implies native/built-in, which means there's never going to be an external wallet config file.  So, yes, I think `seeded` and the token condition are what we're after.